src/components/follow.tsx

```diff
@@ -25,6 +25,28 @@ const styles = (kicker: string, inverted: string): SerializedStyles => css`
 	border: none;
 	background: none;
 	margin-left: 0;
+	margin-top: 4px;
+
+	svg {
+		width: 24px;
+		height: 24px;
+		margin-bottom: -6px;
+		circle {
+			fill: ${kicker};
+		}
+		path {
+			fill: #fff;
+		}
+
+		${darkModeCss`
+		circle {
+			fill: ${inverted};
+		}
+		path {
+			fill: #121212;
```

Comment:
This is `neutral.7` in Source. I'm not sure there's a semantic colour name (`background.inverse` doesn't really work for SVG paths 😬).